config/default.yml

```diff
@@ -98,6 +98,14 @@ Rails/ApplicationRecord:
   VersionAdded: '0.49'
   VersionChanged: '2.5'
 
+Rails/ArelStar:
+  Description: 'Enforces `Arel.star` instead of `"*"` for expanded columns.'
+  Enabled: true
+  Safe: false
+  SafeAutoCorrect: false
+  AutoCorrect: false
```

Comment:
Is there any reason to set this?

Reply:
I set this to not autocorrect by default because it is unsafe.

Reply:
IMO, I think this setting is unnecessary due to `Safe: false` is not auto-corrected by default `rubocop -a` safe option. (In that case, users will explicitly use `rubocop -A` unsefe option.)